### DISCUSSION TITLE
New multi-server, multi-partition web client

### DISCUSSION THREAD

**Discussion by faceless2** [MusicPlayerDaemon/MPD#1739] [Show and tell]
This might be of interest to some: https://github.com/faceless2/mpdqtunes

For anyone developing their own clients; ignore the UI, it's also very thin websocket interface onto one (or more) MPD servers which you're welcome to clone as a basis for other projects.

**Comment by jcorporation** [comment]
Has myMPD not fit your needs? I am interested why.
